src/physics/material/detail/Utils.cc

```diff
@@ -0,0 +1,70 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file Utils.cc
+//---------------------------------------------------------------------------//
+#include "Utils.hh"
+
+#include <algorithm>
+#include <cmath>
+#include "base/Constants.hh"
+#include "base/Quantity.hh"
+#include "physics/material/ElementDef.hh"
+
+namespace celeritas
+{
+namespace detail
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Calculate inverse of Tsai radiation length for bremsstrahlung [cm^2/g].
+ *
+ * See \S33.4.2, pp.452--453, from Review of Particle Physics, calculation of
+ * 1/X0. This is a
+ * semi-empirical formula. The result is like an inverse mass attenuation
+ * coefficient: it must be multiplied by density to produce a length.
+ */
+real_type calc_mass_rad_coeff(const ElementDef& el)
+{
```

Comment:
Any particular reason on having this calculated, considering we're exporting it? (`geant-exporter.cc:258: element.radiation_length_tsai = g4element->GetfRadTsai() / cm;`)

Reply:
Well, partly it's because I forgot that your exporter was including it 😅 

But also I was looking up for the Bremsstrahlung model what the hell the "radlen" was, and then had trouble telling what the quantity was or what its units were. If I hadn't gone through the actual process of implementing this from the source material, I wouldn't have understood it, and I probably would have gotten the units wrong. The quantity is `cm` for material and `g/cm^2` for element -- they're two entirely different quantities!

 I think as time goes on we'll want to move toward more "fundamental" quantities and calculate the "derived" values ourselves.

Reply:
Oh my, I'll fix the unit on my side. 

> I think as time goes on we'll want to move toward more "fundamental" quantities and calculate the "derived" values ourselves.

Ok, so I guess we can validate it and decide whether we use it or just read the data from the exporter in the short run.